src/ResXManager.Infrastructure/ResXManager.Infrastructure.csproj

```diff
@@ -10,13 +10,13 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Fody" PrivateAssets="all" Version="6.8.0" />
-    <PackageReference Include="HtmlAgilityPack" PrivateAssets="all" Version="1.11.51" />
-    <PackageReference Include="ILMerge.Fody" Version="1.24.0" PrivateAssets="all" />
-    <PackageReference Include="JetBrains.Annotations" PrivateAssets="all" Version="2023.2.0" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="all" />
-    <PackageReference Include="Newtonsoft.Json" Version="[9.0.1]" />
-    <PackageReference Include="PropertyChanged.Fody" PrivateAssets="all" Version="4.1.0" />
-    <PackageReference Include="TomsToolbox.Composition" Version="2.8.10" />
+    <PackageReference Include="Fody" PrivateAssets="all" />
+    <PackageReference Include="HtmlAgilityPack" PrivateAssets="all" />
+    <PackageReference Include="ILMerge.Fody" PrivateAssets="all" />
+    <PackageReference Include="JetBrains.Annotations" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="all" />
+    <PackageReference Include="Newtonsoft.Json" />

```

Comment:
`Newtonsoft.Json` must stick to the different versions in the respective projects; for .VSIX x86 it must not be higher than 9.0.1, for .VSIX x64 it must be 13.01; only for .exe we are free to use the latest version.

That would be hard to solve with CPM

Reply:
Fixed with `VersionOverride`.

You can check the PR.